integrations/wootric/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@segment/analytics.js-integration-wootric",
   "description": "The Wootric analytics.js integration.",
-  "version": "2.2.0",
+  "version": "2.3.0",
```

Comment:
@marinhero this is why I used that version number

Reply:
I still think it's weird that the version is higher than the one in the master branch. I think you can keep the version of the integration as `2.3.0` but the beacon should be the same one here than in `master`

Reply:
Ok, so we should always use the same version as the beacon?

Reply:
I think I'm complicating this. Please help me understand, in my mind, the integration has its own version and the Wootric JS SDK has its own version... correct? If so:

- Integration: 2.3.0
- Wootric SDK: x.y.z from master

Where do we use the integration version? is it passed to overwrite the Wootric SDK version?